packages/angular/projects/clr-angular/src/emphasis/_variables.label.scss

```diff
@@ -10,7 +10,7 @@ $clr-label-borderwidth: $clr-global-borderwidth !default;
 // COLOR
 // Usage:
 // - ./_labels.clarity.scss
-$clr-label-font-color-light: $clr-color-neutral-700 !default;
```

Comment:
These changes also need to be made in `./_properties.label.scss`

Reply:
Done.